### DISCUSSION TITLE
合成数据的研究方向

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-wonderland#402] [1️⃣ 新鲜事]
有很多相关的文章和论文，可以看看：

- Machine Learning for Synthetic Data Generation A Review
- [合成数据：探索其方法、应用和挑战](https://medium.com/dsaid-govtech/synthetic-data-navigating-its-methodologies-applications-and-challenges-9af7e5e4208d)
- Exploring Synthetic Data for Artificial Intelligence and Autonomous Systems

算力集群 --> 大模型合成数据生成 --> 数据资产化 --> 数据入表与变现

**Comment by bifenglin** [comment]
其中langchain已经完成了部分的合成数据的应用框架，可以简易的coding就可以使用LLM的能力进行合成数据，相关地址如下，可以简化研究人员研究工作：

https://python.langchain.com/docs/use_cases/data_generation/

**Reply by will-ww** [reply]
这个相当不错~
